README.md

```diff
@@ -1,57 +1,73 @@
-# Setup
+## roachprod
 
-Make sure you have gcloud [installed](https://cloud.google.com/sdk/downloads) and configured (`gcloud auth list` to check, `gcloud auth login` to authenticate). You may want to update old installations (`gcloud components update`).
+⚠️ roachprod is an **internal** tool for testing CockroachDB clusters. Use at
+your own risk! ⚠️
+
+## Setup
+
+Make sure you have [gcloud installed] and configured (`gcloud auth list` to
+check, `gcloud auth login` to authenticate). You may want to update old
+installations (`gcloud components update`).
+
+To build and install into `$GOPATH/bin`:
 
-Building:
 ```
-$ go get github.com/cockroachlabs/roachprod
+$ go get -u github.com/cockroachlabs/roachprod
 ```
 
-# Summary
+## Summary
 
-* clusters are created under the [cockroach-ephemeral](https://console.cloud.google.com/home/dashboard?project=cockroach-ephemeral) GCE project.
-* anyone can connect to any port on VMs in `cockroach-ephemeral` **DO NOT STORE SENSITIVE DATA**.
-* cluster names are prefixed with the user creating them (eg: `roachprod create test` creates the `marc-test` cluster)
-* VMs have a default lifetime of 12h (changeable with the `-l` flag) and are deleted 6 hours after expiration.
-* default settings create 4 VMs (`-n 4`) with 4 CPUs, 15GB ram (`--machine-type=n1-standard-4`) using local SSDs (`--local-ssd`).
+* Clusters are created under the [cockroach-ephemeral] GCE project.
+* Anyone can connect to any port on VMs in `cockroach-ephemeral`.
+  **DO NOT STORE SENSITIVE DATA**.
+* Cluster names are prefixed with the user creating them. For example,
+  `roachprod create test` creates the `marc-test` cluster.
+* VMs have a default lifetime of 12 hours (changeable with the `-l` flag) and
+  are deleted 6 hours after expiration.
+* Default settings create 4 VMs (`-n 4`) with 4 CPUs, 15GB memory
+  (`--machine-type=n1-standard-4`), and local SSDs (`--local-ssd`).
 
-# Setting up a cluster using roachprod/roachperf
+## Cluster quick-start using roachprod/roachperf
 
 ```bash
-# Create cluster with 4 nodes and local SSD (last node is used as load generator by roachperf)
+# Create a cluster with 4 nodes and local SSD.
+# The last node is used as load generator by roachperf.
 export NAME="test"
 export FULLNAME="${USER}-${NAME}"
 roachprod create ${NAME} -n 4 --local-ssd
```

Comment:
This should be `roachprod create ${FULLNAME}`. Mea culpa for not updating this documentation when I made that change. Can you fix the use of `${NAME}` here and below?